tests/test_indexing.py

```diff
@@ -1,33 +1,34 @@
 """ Tests for sh channel indexing"""
```

Comment:
should be put these tests to ``test_spherical``

Reply:
I've moved all tests for indexing (that is also acn related things) to `test_indexing`, to make this a bit more consistent